src/main/java/com/softserve/skillscope/mapper/TalentMapper.java

```diff
@@ -0,0 +1,9 @@
+package com.softserve.skillscope.mapper;
+
+import com.softserve.skillscope.talent.model.entity.Talent;
+import com.softserve.skillscope.talent.model.entity.TalentFlashcard;
+import org.mapstruct.Mapper;
+@Mapper
```

Comment:
If you use the new build and plugins, add this annotation to prevent autowired error
```@Mapper(componentModel = "spring", unmappedTargetPolicy = IGNORE)```